salt/utils/gitfs.py

```diff
@@ -1611,6 +1611,11 @@ class Dulwich(GitProvider):  # pylint: disable=abstract-method
     '''
     def __init__(self, opts, remote, per_remote_defaults,
                  override_params, cache_root, role='gitfs'):
+        salt.utils.warn_until(
+            'Oxygen',
```

Comment:
Wouldn't this be Nitrogen since #39210 removes Dulwich support in develop right now?